MultiVersions/Forge/main/integrateddynamics/lang/zh_tw.json

```
@@ -0,0 +1,1873 @@
+    "block.integrateddynamics.crystalized_menril_block": "門瑞歐結晶方塊",
+    "block.integrateddynamics.crystalized_menril_brick": "門瑞歐結晶磚",
+    "block.integrateddynamics.block_menril_resin": "門瑞歐樹脂",
+    "block.integrateddynamics.drying_basin": "乾燥盆",
+    "block.integrateddynamics.drying_basin.info": "將物品和液體晾乾",
+    "block.integrateddynamics.squeezer": "擠壓架",
+    "block.integrateddynamics.squeezer.info": "站在上面反覆跳躍，利用重力將物品擠壓、壓碎或壓平。使用紅石信號來重置。",
+    "block.integrateddynamics.menril_torch": "門瑞歐火把",
+    "block.integrateddynamics.menril_torch_stone": "門瑞歐石柄火把",
+    "block.integrateddynamics.invisible_light": "不可見的光",
+    "block.integrateddynamics.menril_door": "門瑞歐木門",
+    "block.integrateddynamics.menril_planks_stairs": "門瑞歐木階梯",
+    "block.integrateddynamics.crystalized_menril_block_stairs": "門瑞歐結晶階梯",
+    "block.integrateddynamics.crystalized_menril_brick_stairs": "門瑞歐結晶磚階梯",
+    "block.integrateddynamics.delay": "延遲器",
+    "block.integrateddynamics.delay.info": "將變數前段時間的值公開作為項目。在合成台上清空。",
+    "block.integrateddynamics.block_liquid_chorus": "歌萊果漿",
+    "block.integrateddynamics.crystalized_chorus_block": "歌萊果漿結晶方塊",
+    "block.integrateddynamics.crystalized_chorus_brick": "歌萊果漿結晶磚",
+    "block.integrateddynamics.crystalized_chorus_block_stairs": "歌萊果漿結晶階梯",
+    "block.integrateddynamics.crystalized_chorus_brick_stairs": "歌萊果漿結晶磚階梯",
+    "block.integrateddynamics.mechanical_squeezer": "電動擠壓架",
+    "block.integrateddynamics.mechanical_squeezer.info": "利用電力將物品擠壓、壓碎或壓平",
+    "block.integrateddynamics.mechanical_drying_basin": "電動乾燥盆",
+    "block.integrateddynamics.mechanical_drying_basin.info": "用電力將物品和液體晾乾",
+    "block.integrateddynamics.crystalized_menril_block_slab": "結晶門瑞歐半磚",
+    "block.integrateddynamics.crystalized_menril_brick_slab": "結晶門瑞歐磚半磚",
+    "block.integrateddynamics.crystalized_chorus_block_slab": "歌萊果漿結晶半磚",
+    "block.integrateddynamics.crystalized_chorus_brick_slab": "歌萊果漿結晶磚半磚",
+
+    "_comment": "Items",
+    "item.integrateddynamics.general.id": "§e§lID：§r%s",
+    "item.integrateddynamics.wrench": "板手",
+    "item.integrateddynamics.wrench.info": "Shift + 右鍵以切換模式",
+    "item.integrateddynamics.wrench.mode": "§e模式：§r%s",
+    "item.integrateddynamics.wrench.mode.default": "預設",
+    "item.integrateddynamics.wrench.mode.default.info": "拆下設備或式旋轉方塊",
+    "item.integrateddynamics.wrench.mode.offset": "保存偏移量",
+    "item.integrateddynamics.wrench.mode.offset.info": "Shift + 右鍵一個位置，再右建設配來設定偏移量。",
+    "item.integrateddynamics.wrench.mode.offset.saved": "保存位置：%s",
+    "item.integrateddynamics.wrench.mode.offset.success": "已設定設備的偏移位置",
+    "item.integrateddynamics.wrench.mode.offset.incomplete": "板手還沒設定位置",
+    "item.integrateddynamics.wrench.mode.offset.fail": "無法設定偏移量，需要強化以提升最大偏移量。",
+    "item.integrateddynamics.wrench.mode.offset.pos": "位置：%s",
+    "item.integrateddynamics.wrench.mode.offset_side": "保存偏移量和方位",
+    "item.integrateddynamics.wrench.mode.offset_side.info": "Shift + 右鍵一個位置，再右建設配來設定偏移量和方位。",
```

原文為按下**Shift**，但實際上模組是識別**潛行**，如此翻譯較能避免誤導。
```suggestion
    "item.integrateddynamics.wrench.mode.offset_side.info": "潛行 + 按下右鍵一個位置，再右建設配來設定偏移量和方位。",
```

---

```suggestion
    "item.integrateddynamics.wrench.mode.offset_side.info": "Shift + 右鍵點擊一個位置，再對著設備點擊右鍵來設定偏移量和方位。",
```